shell_sort/Python/shell_sort.py

```diff
@@ -0,0 +1,34 @@
+def shell_sort(arr):
+    """
+    Fuction to sort using Shell Sort
+    <https://en.wikipedia.org/wiki/Shellsort>.
+
+    :param arr:     A list of element to sort
+    """
+
+    gap = int((len(arr)/2))
+    while gap > 0: 
+        for i in range (gap, len(arr)):
+            temp = arr[i]
+            j = i
+            while j >= gap and arr[j - gap] > temp:
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpxfzu3aem/shell_sort/Python/shell_sort.py
+++ b/tmp/tmpxfzu3aem/shell_sort/Python/shell_sort.py
@@ -7,8 +7,8 @@
     """
 
     gap = int((len(arr)/2))
-    while gap > 0: 
-        for i in range (gap, len(arr)):
+    while gap > 0:
+        for i in range(gap, len(arr)):
             temp = arr[i]
             j = i
             while j >= gap and arr[j - gap] > temp:
```